tests/wpt/mozilla/tests/mozilla/bluetooth/getCharacteristics/disconnect-invalidates-objects-with-uuid.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
```

Comment:
Given how similar this test and disconnect-invalidates-objects.html are, it would be nice to combine the with/without uuids tests in the same file rather than duplicating so much code in each of them.